code/modules/uplink/uplink_items/stealthy.dm

```diff
@@ -99,14 +99,3 @@
 	cost = 10
 	surplus = 50
 	purchasable_from = ~UPLINK_ALL_SYNDIE_OPS
-
-/datum/uplink_item/stealthy_weapons/contrabaton
-	name = "Contractor Baton"
-	desc = "A compact, specialised baton assigned to Syndicate contractors. Applies light electrical shocks to targets. \
-	These shocks are capable of affecting the inner circuitry of most robots as well, applying a short stun. \
-	Has the added benefit of affecting the vocal cords of your victim, causing them to slur as if inebriated."
-	item = /obj/item/melee/baton/telescopic/contractor_baton
-	cost = 7
-	surplus = 50
-	limited_stock = 1
-	purchasable_from = UPLINK_TRAITORS | UPLINK_SPY
```

Comment:
argue for its removal from spies if you are removing it from them as well

Reply:
spies is luck though isnt it?

Reply:
oh wait i see hold on im dumb